l10n_nl_bsn/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================================
+Burgerservicenummer (BSN) for Partners
+======================================
+
+This module adds the BSN (Burgerservicenummer) field on partner forms.
+
+The field is visible when the flag is_company is false.
+
+A double check on the BSN is done when inserting/modifying its value:
+ - validation of the BSN (check whether the format is correct);
+ - check if another partner with the same BSN already exists.
+
+In both cases, a non-blocking alert is shown.
+
+
+Installation
+============
+
+The module depends on the external library 'python-stdnum'.
+
+You can install that library by using pip3:
+
+* pip3 install python-stdnum
+
+
+Configuration
+=============
+
+For security reasons the BSN number should be only visible to HR related roles.
+Otherwise this will be in violation to the security framework of WBP regarding
+the protection of persons info.
+
+To be able to see the BSN number, give the proper permits to the user:
+
+* User must belong to the "HR Officer" group
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Open a form of a contact, eg.: a person (uncheck the flag "Is a Company?")
```

Comment:
As above, this line is not accurate anymore

Reply:
Done in https://github.com/OCA/l10n-netherlands/pull/199/files#diff-de53f2d9eb0ad4e03a14910110c926dfR3